packages/components/src/components/Checkbox/Checkbox.test.tsx

```diff
@@ -0,0 +1,155 @@
+import 'react-native'
+import { fireEvent, render, screen } from '@testing-library/react-native'
+import React from 'react'
+
+import { Checkbox } from './Checkbox'
+import { Icon } from '../Icon/Icon'
+
+const mockedColorScheme = jest.fn()
+
+jest.mock('react-native/Libraries/Utilities/useColorScheme', () => {
+  return {
+    default: mockedColorScheme,
+  }
+})
+
+describe('Checkbox', () => {
+  const onPressSpy = jest.fn()
+
+  const commonProps = {
+    header: 'Header text',
+    description: 'Description text',
+    hint: 'Hint text',
+    label: 'Label text',
+    testID: 'testCheckbox',
+    onPress: onPressSpy,
+  }
+
+  const errorMsg = 'Error text'
+
+  const getIconName = async () => {
+    const icon = await screen.root.findByType(Icon)
+    return icon.props.name
+  }
```

Comment:
Wonder if we should make a ticket for a utils file for test functions since, while we have two different variations (getIcon and getIconName), this is the 6th test suite using a generic getIcon function.

Not sure if we could similarly pull out color scheme mocking that every test uses, but might be worth exploring.

Reply:
Created https://github.com/department-of-veterans-affairs/va-mobile-library/issues/539